packages/gestalt/src/moduleTypes.js

```diff
@@ -2,44 +2,18 @@
 import { type Node } from 'react';
```

Comment:
I reworked this file a bit, colocating types with their components when possible (for easier reference) and restricting the types in this file to only the ones shared across components